src/extensions/events.py

```diff
@@ -0,0 +1,24 @@
+import discord
+from discord.ext import commands
+import re
+
+class Events:
+  def __init__(self, bot):
+    self.bot: commands.Bot = Bot
+  
+  @commands.event
+  async def on_message(self, message):
```

Comment:
At present, this coroutine is not very extendable. Let's have a discussion on the best way to make this more maintainable